quantization/test/test_optimal_quantization.py

```diff
@@ -0,0 +1,84 @@
+"""
+Unit test for optimal quantization
+"""
+
+import unittest
+
+import numpy as np
+import quantization.optimal_quantization as oq
+
+from geomstats.hypersphere import Hypersphere
+from geomstats.hypersphere import HypersphereMetric
+
+TOLERANCE = 5e-3
+
+
+class TestOptimalQuantization(unittest.TestCase):
+    _multiprocess_can_split_ = True
+
+    def setUp(self):
+        np.random.seed(1234)
+        self.space = 'S2'
+        self.metric = HypersphereMetric(dimension=2)
+        self.n_points = 1000
+        self.points = Hypersphere(dimension=2).random_von_mises_fisher(
+                kappa=10,
+                n_samples=self.n_points)
+
+    def test_sample_from(self):
+        """
+        Check that the sample is one of points.
+        """
+        sample = oq.sample_from(self.points)
+        result = False
+        for i in range(self.n_points):
+            if (self.points[i, :] == sample).all():
```

Comment:
Could you use `np.allclose(self.points[i, :], sample)`?

Reply:
Corrected.